### PR TITLE
[fix](be) avoid core dump caused by Log(FATAL) in doris_main.cpp

### PR DESCRIPTION
**Pull request by Tech-Circle-48**
## Proposed changes

Issue Number: 31441

Solved core dump caused by Log(FATAL)

## Further comments

LOG(FATAL) in Doris_main.cpp changed

### CODE REVIEW

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Review by mrhhsg — APPROVED** [review]

File: be/src/service/doris_main.cpp [hunk 1]
```diff
@@ -166,8 +166,8 @@ auto instruction_fail_to_string(InstructionFail fail) {
     case InstructionFail::ARM_NEON:
         ret("ARM_NEON");
     }
-    LOG(FATAL) << "__builtin_unreachable";
```
Comment (yiguolei) [inline be/src/service/doris_main.cpp:169]: why not core dump? if returned, then it will core in other place.

Comment (Tech-Circle-48) [reply be/src/service/doris_main.cpp:169]: now,how do i resolve the issue

**Pull request closed without merging**